new-website/utils/tutorials/build_pdf_book.py

```diff
@@ -182,6 +200,6 @@ def compile_information_pages():
     html_to_pdf()
     merge_pdf()
     compile_information_pages()
-    merge_pdf_pages(['storage/title.pdf', 'storage/acknowledgement.pdf', 'storage/contents.pdf', 'storage/merged.pdf'])
-    upload_file('storage/full_pdf.pdf', 'deepchemtutorials', 'TutorialsBook.pdf')
+    merge_pdf_pages(['cover.pdf', 'storage/title.pdf', 'storage/acknowledgement.pdf', 'storage/contents.pdf', 'storage/merged.pdf'])
```

Comment:
@Cannon07 Actually looking at this, there is no date stamp in the name. Please fix this before merging